src/sentry/static/sentry/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -208,12 +249,37 @@ class AbstractIntegrationDetailedView<
     );
   }
 
-  //Returns the content shown in the top section of the integration detail
-  renderTopSection() {
+  renderAddInstallButton(configurations = true) {
     const {organization} = this.props;
-
     const {IntegrationDirectoryFeatures} = getIntegrationFeatureGate();
+
+    return (
+      <IntegrationDirectoryFeatures {...this.featureProps}>
+        {({disabled, disabledReason}) => (
+          <DisableWrapper>
+            <Access organization={organization} access={['org:integrations']}>
+              {({hasAccess}) => (
+                <Tooltip
+                  title={t(
+                    'You must be an organization owner, manager or admin to install this.'
+                  )}
+                  disabled={hasAccess}
+                >
+                  {this.renderButton(disabled, hasAccess, configurations)}
```

Comment:
@NisanthanNanthakumar I don't think having a new function here makes sense since it's only being used in one place and it's pretty simple. Getting to have too many functions that are basically "render button". What about:
```
{!configurations && disabled ? <div /> : this.renderTopButton(disabled, hasAccess);}
```